shared/lib/names.js

```diff
@@ -91,34 +105,29 @@ function packageResolveToPath(packageEntry, { modulePath, moduleRoot } = {}) {
  *
  * {@link https://github.com/alphagov/govuk-frontend/issues/3755}
  *
- * @example
- * Resolving components relative to a default package entry
- *
- * - GOV.UK Frontend v4 './govuk/components/accordion/accordion.mjs'
- * - GOV.UK Frontend v5 './dist/govuk/components/accordion/accordion.mjs'
- *
- * ```mjs
- * const templatePath = packageResolveToPath('govuk-frontend', {
- *   modulePath: `components/accordion/accordion.mjs`
- * })
- * ```
  * @param {string} packageName - Installed npm package name
  * @param {PackageOptions} [options] - Package resolution options
  * @returns {string} Path to installed npm package field
  */
-function packageTypeToPath(
-  packageName,
-  { modulePath, moduleRoot, type = 'commonjs' } = {}
-) {
+function packageTypeToPath(packageName, options = {}) {
+  const { modulePath, moduleRoot, type = 'commonjs' } = options
+
+  // Assume package.json is always resolvable
```

Comment:
**question** Can a package prevent access to its `package.json`? If not, can we drop the "Assume", please, to avoid confusion when we read back this code in the future.

Reply:
Yeah sadly it can, so this is the correct comment 😔

With package exports you can quite easily [lock yourself out from **package.json** access](https://github.com/isaacs/node-glob/pull/503#issuecomment-1450874010)

So you'll often see it as a separate entry point:
https://github.com/isaacs/tshy?tab=readme-ov-file#exports